build/rocm/run_single_gpu.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
```

Comment:
All .py files have to have the apache license header. (Possibly other files also, but certainly the .py files.)

Same everywhere.

Reply:
Done.